misc/python/materialize/checks/materialized_views.py

```diff
@@ -0,0 +1,68 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
This is `misc/python/materialize/checks/recorded_views.py` with all occurrences of "recorded view" replaced.